apps/modernization-ui/src/components/selection/multi/MultiSelectInput.tsx

```diff
@@ -70,7 +71,16 @@ export const MultiSelectInput = ({
             onChange(values);
         }
     };
-    const Input = (props: any) => <components.Input {...props} maxLength={50} />;
+
+    const handleInputChange = (searchText: string, action: { action: string }) => {
+        if (action.action !== 'input-blur' && action.action !== 'set-value') {
+            setSearchText(searchText);
+        }
+    };
+
+    const filteredOptions = useMemo(() => {
+        return selectableOptions.filter((option) => option.label.toLowerCase().includes(searchText.toLowerCase()));
+    }, [selectableOptions, searchText]);
```

Comment:
question: Is the react-select component taking care of this filtering already?  This might not be needed.